src/client/app/operators.ts

```diff
@@ -1,7 +1,7 @@
-// rxjs
-import 'rxjs/add/observable/throw';
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/catch';
+// rxjs: Add only if lettable operators are being used. Pipeable operators don't require this.
```

Comment:
Would you drop this file completely? We don't want to encourage people to write imports with side effects.

Reply:
Sure.

Btw, When we use `import { catchError } from 'rxjs/operators/catchError';`, it gets bundled just fine. But when we use `import { catchError } from 'rxjs/operators';`, we get all the operations in the prod bundle. There's an easy webpack solution for that, but do you think it is possible with SystemJS?